Source/Mosa.Korlib/System/BitConverter.cs

```diff
@@ -19,81 +19,74 @@ public static byte[] GetBytes(bool value)
 		// Converts a char into an array of bytes with length two.
 		public static byte[] GetBytes(char value)
 		{
-			byte[] bytes = new byte[sizeof(char)];
-			Unsafe.As<byte, char>(ref bytes[0]) = value;
-			return bytes;
+			return GetBytes((short)value);
```

Comment:
prefer the use of Unsafe helper class instead

Reply:
Check.